### PR TITLE
add new html and css code

### PR DESCRIPTION
**Pull request by den-cloud**
In this project I was reviewing my project from the trails and add necessary, to have my project pass all no linter errors requirement.

I used html and css to build this project.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -5,11 +5,13 @@ body {
 }
 
 #header {
+  top: 0em;
```
Comment (den-cloud) [inline css/style.css]: css/style.css
 8:9  ✖  Unexpected unit   length-zero-no-unit
Try :
padding-top: use cm or rem;

File: css/style.css [hunk 2]
```diff
@@ -5,19 +5,21 @@ body {
...
   background-color: gray;
+  align-items: center;
 }
 
 .nav-link {
   text-decoration: none;
   color: white;
   list-style: none;
   display: flex;
-  padding: 0.5em;
```
Comment (den-cloud) [inline css/style.css:20]: Invalid value for `padding`        csstree/validator
Unexpected whitespace before ";"   declaration-block-semicolon-space-before

Add  a value between padding: and ;
example: padding: 1em;

**Review by den-cloud — COMMENTED** [review]
I added new html and css code.
In the project , I added folder called gitgub/workflows
added files linters.yml, gitignore, hintrc, stylelintrc and a read.md

run npx hint to find and correct errors
run npx stylelint to and correct erros

**Review by den-cloud — COMMENTED** [review]
I added new html and css code.
In the project , I added folder called gitgub/workflows
added files linters.yml, gitignore, hintrc, stylelintrc and a read.md

run npx hint to find and correct errors
run npx stylelint to and correct erros

**Review by mimipeshy — APPROVED** [review]
## Hi @den-cloud 👋🏾  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Summarizing your reviews using words for example `i think you should try and increasing padding-top value to 12cm. You can either use cm or rem`


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**